src/cl_commands.cpp

```diff
@@ -282,6 +282,22 @@ void CLIENTCOMMANDS_Say( ULONG ulMode, const char *pszString )
 	CLIENT_GetLocalBuffer( )->ByteStream.WriteString( chatstring );
 }
 
+//*****************************************************************************
+//
+void CLIENTCOMMANDS_PrivateSay( ULONG ulPlayer, const char *pszString )
```

Comment:
This function shares a lot of code with `CLIENTCOMMANDS_Say`. I think it would be better if `CLIENTCOMMANDS_Say` is extended to cover the case that `ulMode` is `CHATMODE_PRIVATE_SEND`. For this it would need an additional optional argument `ulPlayer` that defaults to `MAXPLAYERS`.